docker-compose.yml

```diff
@@ -21,27 +22,64 @@ secrets:
     file: ./secrets/geoserver_privileged_user_passwd.txt
 
 services:
+  copy-datadir:
+    image: alpine
+    command: sh -c "rm -r /etc/georchestra/* && cp -r -f -v /mnt/datadir/* /etc/georchestra/ && chmod 777 -R -v /etc/georchestra/" # "sleep 6000"
+    volumes:
+      - ./config:/mnt/datadir
+      - georchestra_datadir:/etc/georchestra
+
+  envsubst:
+    image: georchestra/k8s-initcontainer-envsubst
+    depends_on:
+      copy-datadir:
+        condition: service_completed_successfully
+    environment:
+      - DEBUG=yes
+      - SUBST_FILES=/etc/georchestra/security-proxy/targets-mapping.properties /etc/georchestra/datafeeder/frontend-config.json /etc/georchestra/geonetwork/microservices/ogc-api-records/config.yaml
+    env_file:
+      - envar
```

Comment:
why is there is a need to include the environment file named `envar`?

Reply:
oh yeah i forget this !
because we need all env var, would be complex to add them all one by one